### PR TITLE
initial structure, local development setup description

### PR DESCRIPTION
**Pull request by 1mcodes**
It can be buggy right now. PR only to review the changes.

### CODE REVIEW

**Comment by 1mcodes** [comment]
I've created new files keyboard/games/votes to keep logic as simple as possible.

File: cmd/scrumpoke/main.go [hunk 1]
```diff
@@ -2,187 +2,55 @@ package main
 
 import (
 	"context"
-	"fmt"
+	"github.com/Pawilonek/scrumpoke/internal/config"
+	"github.com/Pawilonek/scrumpoke/internal/poker"
```
Comment (Pawilonek) [inline cmd/scrumpoke/main.go:6]: I would like to sort the imports. But, this could be an issue for the future - #4

File: internal/poker/games.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+type (
+	gameHash string
+
+	games map[gameHash]game
+	game  struct {
+		votes     votes
+		createdAt int
+		updatedAt int
+		creator   *tgbotapi.User
+		jiraIssue string
```
Comment (Pawilonek) [inline internal/poker/games.go:19]: I do not like that it's "jira" we could but any string here or any issue tracker in the future.

**Review by Pawilonek — APPROVED** [review]

File: internal/poker/games.go (hunk 2 above)
Comment (1mcodes) [reply internal/poker/games.go:19]: Sure, we can call it differently

**Pull request merged**